pkg/asset/machines/azure/azuremachines.go

```diff
@@ -140,7 +140,7 @@ func GenerateMachines(platform *azure.Platform, pool *types.MachinePool, userDat
 				NetworkInterfaces: []capz.NetworkInterface{
 					{
 						SubnetName:            subnet,
-						AcceleratedNetworking: ptr.To(mpool.VMNetworkingType == azure.AcceleratedNetworkingEnabled),
+						AcceleratedNetworking: ptr.To(mpool.VMNetworkingType != string(azure.VMNetworkingTypeBasic)),
```

Comment:
That logic is so that we can implement machinepool "inheritance": https://github.com/openshift/installer/blob/master/pkg/asset/machines/master.go#L330-L341. I don't think it needs to be changed for capi/azure feature in this PR.